zebra-rpc/qa/rpc-tests/addnode.py

```diff
@@ -33,6 +33,7 @@ def run_test(self):
 
         # As we connected the nodes to each other, they should have,
         # at least 4 peers. Poll for that.
+        # TODO: Add a function for this check.
```

Comment:
Should the line `timeout -= wait_time` be `timeout_for_connetions -= wait_time`?

```suggestion
        # TODO: Move this check to its own function.
```

Reply:
Changed at https://github.com/ZcashFoundation/zebra/pull/9629/commits/01689d18f439663ff655f0184007c531a931fea1